docs/uvdata_tutorial.rst

```diff
@@ -483,55 +483,20 @@ UVData: Phasing
 ---------------
 Phasing/unphasing data
 
-a) Data with a single phase center.
+a) Data with a multiple phase centers enabled.
```

Comment:
Remove "a"

Reply:
I'm actually just going to remove this whole heading because there's now only one subheading, which feels silly.